pkg/vadmin/restart_node_vc.go

```diff
@@ -62,7 +62,10 @@ func (v *VClusterOps) genStartNodeOptions(s *restartnode.Parms, certs *HTTPSCert
 	opts.UserName = su
 	opts.Password = &v.Password
 	opts.Nodes = s.RestartHosts
-	opts.StatePollingTimeout = v.VDB.GetRestartTimeout()
+	vdbTimeout := v.VDB.GetRestartTimeout()
```

Comment:
Is this related to a specific change in vcluster? Or does this issue exist in 24.2.0. If it's an issue with 24.2.0, you may want to put the change into main first and pick it up in vnext via a merge.

Reply:
* For restart_node, I think the reason comes from my change: https://git.verticacorp.com/projects/VER/repos/server/pull-requests/20821/diff#platform/vcluster/vclusterops/start_node.go?f=149. I moved the default value to func (options *VStartNodesOptions) setDefaultValues(). Thank you for finding this error.

* For start_db, I think it exists in 24.2.0.

Reply:
Because it is start_db, it probably should be in main and a changie entry added. The changie should only say you are fixing start_db. start_node is fine in prior releases.

Reply:
start_db is fine in main. The change to make start_db fail is made last month in this PR https://git.verticacorp.com/projects/VER/repos/server/pull-requests/20685/diff#platform/vcluster/vclusterops/start_db.go?f=111

We don't need to put the change into main.

Reply:
I also suspect this is why the upgrade test fails in my PR